src/components/home/EventTable.tsx

```diff
@@ -11,9 +12,19 @@ function EventTable({ events }: EventTableProps) {
       <TableBody>
         {events.map((event) => (
           <TableRow key={event.id}>
-            <TableCell className='max-w-80 rounded-lg p-2'>
-              <h3 className='truncate'>{event.localization.name}</h3>
-              <p className='truncate'>{event.localization.summary}</p>
+            <TableCell className='max-w-80 rounded-lg p-0'>
+              <Link
+                variant='none'
+                size='none'
+                href={{
+                  pathname: '/events/[eventId]',
+                  params: { eventId: event.id },
+                }}
+                className='block p-2'
```

Comment:
```suggestion
                className='block rounded-xl p-2 focus-visible:ring-inset'
```
Keep the ring inside the link so it's not cut off by the table when tabbing to it.